zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -261,3 +268,5 @@ object Server {
     }
   }
 }
+
+case class ServerChannelOption(channelOption: ChannelOption[Any], value: Any)
```

Comment:
I am getting java type error mismatches while trying to use generic type. Working on a fix for this.